### ISSUE TITLE
Importance score for dataset training samples

### ISSUE THREAD

**Issue by dzeber** [mozilla/PRESC#8]
Implement a way to assess the importance of an inidividual training datapoint to the performance of the model. This could be done by training the same model with a particular point included and then excluded from the training set, and computing the difference in performance scores on the test set.

**Comment by janvi04** [comment]
@dzeber I want to work on this issue. I had a doubt. I an working with winequality dataset. Can I perform this task on the same dataset?
Also, this task requires us to asses the performance of a model with a particular datapoint and without it. Pls correct me if I am wrong.

**Comment by dzeber** [comment]
@janvi04 Yes, you would implement a general method, and you can test it out on any dataset (you can reuse your work from #2). 

> asses the performance of a model with a particular data point and without it

The goal is to come up with a general score for the "importance" of an individual point in the training set. This would be useful in studying misclassifications, eg. to see whether they are "important" or not. You are encouraged to develop your own approach. Training the model with and without the point is one possible approach but there are others.

**Comment by janvi04** [comment]
Thanks @dzeber. I will work on it.

**Comment by tab1tha** [comment]
I was about to make a pull request to this .  Please can you reopen it?

**Comment by mlopatka** [comment]
I accidentally closed this while merging in a PR yesterday. The issue remains open; apologies.

**Comment by tab1tha** [comment]
@mlopatka You have mistakenly closed it again.

**Comment by mlopatka** [comment]
thanks
